prime-router/docs/playbooks/shutdown.md

```diff
@@ -0,0 +1,29 @@
+# Shutdown Options
+
+In the event the project needs to shutdown quickly in the response to an incident, the following options are available:
+
+## Disable a Function
```

Comment:
I think we should also show how to disable the function from the command line using the `az` tool:
```
az functionapp config appsettings set --name <FUNCTION_APP_NAME>
                                      --resource-group <RESOURCE_GROUP_NAME>
                                      --settings AzureWebJobs.<FUNCTION_NAME>.Disabled=true
```
Note that you are setting the `disabled` state to true, instead of setting an enabled state to false (or in other words: to re-enable, you set `<stuff>.Disabled=false`).

We can refer to https://docs.microsoft.com/en-us/azure/azure-functions/disable-function as well which has additional mechanisms

Reply:
Done!